src/test/unit/AllocationManagerUnit.t.sol

```diff
@@ -2175,53 +2254,53 @@ contract AllocationManagerUnitTests_SetAllocationDelay is AllocationManagerUnitT
     }
 
     function test_fuzz_setDelay_multipleTimesWithinConfigurationDelay(
-        uint32 firstDelay, uint32 secondDelay
-    ) public {
-        firstDelay = uint32(bound(firstDelay, 1, type(uint32).max));
-        secondDelay = uint32(bound(secondDelay, 1, type(uint32).max));
+        Randomness r
+    ) public rand(r) {
+        uint32 firstDelay = uint32(r.Uint256(1, type(uint32).max));
```

Comment:
Can we change this to 0 since allocationDelay of 0 is enabled now. These bounds were missed previously in that PR

Reply:
[61772f2](https://github.com/Layr-Labs/eigenlayer-contracts/pull/860/commits/61772f2f930aa2a816f4ee3a1eb6416e505f8596)